### DISCUSSION TITLE
Will this work in a GCC Tenant?

### DISCUSSION THREAD

**Discussion by dfoelker** [OfficeDev/TACO#26] [Q&A]
I am having many issues during the deployment of the Power App and flows.  So I am wondering if this will even work in a GCC tenant?  Has anyone tried?  GCC uses https://make.gov.powerapps.us/ vs. https://make.powerapps.com/, etc. 

https://learn.microsoft.com/en-us/power-platform/admin/powerapps-us-government#power-apps-us-government-service-urls

**Comment by JohanVeldhuis** [comment]
Hi 

@dfoelker sorry for the late reply. We didn't test it yet in GCC tenants and I do think it might cause some issues not specifically in the PowerApps but more on the Teams PowerShell end. The reason for this is that we currently don't specify the TeamsEnvironmentName parameter which I do think is required when connecting to GCC environments.

**Reply by dfoelker** [reply]
Hello Johan.  Is this something that could be added?  I have a GCC environment that can be used for testing.
